api/AudioScheduledSourceNode.json

```diff
@@ -11,7 +11,7 @@
             {
               "version_added": "14",
               "version_removed": "56",
-              "notes": "Interface implemented as AudioSourceNode."
+              "alternative_name": "Interface implemented as AudioSourceNode."
```

Comment:
Alternative_name is not a sentence; currently it doesn't even appear as a note in the generated table on mdn, but this name is used to build as structure like: 14-56 as AudioSourceNode.

So "alternative_name": "AudioSourceNode".